packages/flutter/test/services/raw_keyboard_test.dart

```diff
@@ -1350,7 +1350,7 @@ void main() {
           'type': 'keydown',
           'keymap': 'web',
           'code': 'RandomCode',
-          'key': null,
+          'key': '0',
```

Comment:
I wasn't sure what to do with these. The key event data requires these are non-null

Reply:
You can just remove the entire line.  I think I just put in the nulls to highlight that for the test they weren't being specified. A comment would have been better.

Reply:
Hmm, maybe the signatures got updated? Last I checked it had to be non-null, but this PR is a week or so old now